index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/style.css" />
```

Comment:
- [ ] Your styles are not called properly, instead of `href="/style.css"` you should add a dot "." before "/".

What you have:
```
<link rel="stylesheet" href="/style.css" />
```
What you should have:
```
<link rel="stylesheet" href="./style.css" />
```